### DISCUSSION TITLE
what do the min delivery time and max delivery time fields correspond to ? (just below the description)

### DISCUSSION THREAD

**Discussion by nabilfatfat** [mirumee/saleor-dashboard#2292] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/112693134/188863520-a9fe39dc-ba56-40af-bcdb-722d580e6157.png)

**Comment by peelar** [comment, marked as answer]
Hello @nabilfatfat 👋 

The fields "Min Delivery Time" and "Max Delivery Time" stand for the estimated timeframe for the delivery (e.g. "the delivery should take from 1 to 4 days").

**Reply by nabilfatfat** [reply]
thank you @peelar ! I was hoping it defines when it is possible for a user to pass an order but I guess I will have to work on a local app for this feature
Have a good day
